CondFormats/HcalObjects/interface/HcalQIEType.h

```diff
@@ -12,6 +12,7 @@ contains the QIE Typese + corresponding DetId
 class HcalQIEType
 {
  public:
+  enum {QIE8=0, QIE10=1, QIE11=2};
```

Comment:
It should also be noted that QIEType was only added to 80X ~2 months ago in #12473, so it really hasn't yet been used anywhere centrally and minor modifications at this stage won't cause any persistency problems.